installer/roles/image_build/tasks/main.yml

```diff
@@ -53,7 +53,7 @@
       http_proxy: "{{ http_proxy | default('') }}"
       https_proxy: "{{ https_proxy | default('') }}"
       no_proxy: "{{ no_proxy | default('') }}"
-    path: "image_build/files"
+    path: "roles/image_build/files"
```

Comment:
We should probably do `path: "{{ role_path }}/files"`

Reply:
I concur. Fixed